seal_rookery/__init__.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 
-class _SealRookery(object):
+class _SealRookery:
```

Comment:
This one was done by [object](https://docs.astral.sh/ruff/rules/useless-object-inheritance/)